x-pack/plugins/snapshot_restore/public/app/sections/home/snapshot_list/snapshot_details/snapshot_details.tsx

```diff
@@ -63,56 +63,53 @@ const SnapshotDetailsUi: React.FunctionComponent<Props> = ({
   let content;
 
   if (snapshotDetails) {
-    const { indexFailures } = snapshotDetails;
-
-    if (indexFailures.length) {
-      const tabOptions = [
-        {
-          id: TAB_SUMMARY,
-          name: (
-            <FormattedMessage
-              id="xpack.snapshotRestore.snapshotDetails.summaryTabTitle"
-              defaultMessage="Summary"
-            />
-          ),
-          testSubj: 'srSnapshotDetailsSummaryTab',
-        },
-        {
-          id: TAB_FAILURES,
-          name: (
-            <FormattedMessage
-              id="xpack.snapshotRestore.snapshotDetails.failuresTabTitle"
-              defaultMessage="Failed indices ({failuresCount})"
-              values={{ failuresCount: indexFailures.length }}
-            />
-          ),
-          testSubj: 'srSnapshotDetailsFailuresTab',
-        },
-      ];
-
-      tabs = (
-        <Fragment>
-          <EuiSpacer size="s" />
-          <EuiTabs>
-            {tabOptions.map(tab => (
-              <EuiTab
-                onClick={() => setActiveTab(tab.id)}
-                isSelected={tab.id === activeTab}
-                key={tab.id}
-                data-test-subject={tab.testSubj}
-              >
-                {tab.name}
-              </EuiTab>
-            ))}
-          </EuiTabs>
-        </Fragment>
-      );
-    }
+    const { indexFailures, state } = snapshotDetails;
+    const tabOptions = [
+      {
+        id: TAB_SUMMARY,
+        name: (
+          <FormattedMessage
+            id="xpack.snapshotRestore.snapshotDetails.summaryTabTitle"
+            defaultMessage="Summary"
+          />
+        ),
+        testSubj: 'srSnapshotDetailsSummaryTab',
+      },
+      {
+        id: TAB_FAILURES,
+        name: (
+          <FormattedMessage
+            id="xpack.snapshotRestore.snapshotDetails.failuresTabTitle"
+            defaultMessage="Failed indices ({failuresCount})"
+            values={{ failuresCount: indexFailures.length }}
+          />
+        ),
+        testSubj: 'srSnapshotDetailsFailuresTab',
+      },
+    ];
+
+    tabs = (
+      <Fragment>
+        <EuiSpacer size="s" />
+        <EuiTabs>
+          {tabOptions.map(tab => (
+            <EuiTab
+              onClick={() => setActiveTab(tab.id)}
+              isSelected={tab.id === activeTab}
+              key={tab.id}
+              data-test-subject={tab.testSubj}
+            >
+              {tab.name}
+            </EuiTab>
+          ))}
+        </EuiTabs>
+      </Fragment>
+    );
 
     if (activeTab === TAB_SUMMARY) {
       content = <TabSummary snapshotDetails={snapshotDetails} />;
     } else if (activeTab === TAB_FAILURES) {
-      content = <TabFailures indexFailures={indexFailures} />;
+      content = <TabFailures state={state} indexFailures={indexFailures} />;
```

Comment:
I got a little bit confused with this prop name, thinking that we were passing component state, maybe could be renamed `snapshotState`?

Reply:
Done.